tests/unit/command/test_experiments.py

```diff
@@ -791,3 +791,98 @@ def test_show_experiments_pcp(tmp_dir, mocker):
 
     assert kwargs["output_path"] == str(tmp_dir / "dvc_plots")
     assert kwargs["color_by"] == "Experiment"
+
+
+def test_show_checkpoint_csv(
```

Comment:
A question is that is there any else method to test this except for adding so many lines to create a test case.